### ISSUE TITLE
reduced cell surface expression of class I HLA antigens.

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#10926]
**Preferred term label:**
reduced cell surface expression of class I HLA antigens

**Synonyms**
reduced cell surface expression of class I MHC antigens

**Definition (free text, please give PubMed ID)**
See: https://hpo.jax.org/browse/term/HP:0031389 -- update definitions

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

MHC class I deficiency 3 | 620814

MHC class I deficiency 3 	[620814](https://omim.org/entry/620814)
PMID: 12149238. 1/1 (phenopacket)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
Adding new term: 
- ID: HP:6001190
- label: Abnormal MHC surface expression
- parents: [Abnormality of immune system physiology (HP:0010978)](https://hpo.jax.org/app/browse/term/HP:0010978)
- pmid: 
- def: A deviation from the normal level of major histocompatibility complex (MHC) molecules expressed at the cell surface. The cell-mediated adaptive immune response is regulated by the MHC, which is called human leukocyte antigen (HLA) in humans. HLA molecules are cell surface glycoproteins whose primary function is to present endogenous and exogenous antigens to T lymphocytes for recognition and response. The HLA molecules that present antigen to T lymphocytes are divided into two main classes: HLA class I and HLA class II molecules.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10926
- synonyms: 


TODO -- rearrange and add new terms.

**Comment by pnrobinson** [comment]
added 
HP_0025794 Reduced MHC I cell surface expression
